Core/Core/Files/Model/API/APIFile.swift

```diff
@@ -437,7 +437,7 @@ public struct PostFileUploadRequest: APIRequestable {
         )))
         return form
     }
-    public var isBodyFromURL: Bool { true }
+    public var isBodyFromURL: Bool { false }
```

Comment:
This will break file uploads in case the file is big enough not to fit into memory. This flag indicates if the file is read into memory before uploading or read from a file sequentially to reduce the memory footprint.

Reply:
Added `isBodyFromURL` as a parameter. Please review the changes again.